FirebaseVertexAI/Sources/GenerateContentResponse.swift

```diff
@@ -17,13 +17,28 @@ import Foundation
 /// The model's response to a generate content request.
 @available(iOS 15.0, macOS 11.0, macCatalyst 15.0, *)
 public struct GenerateContentResponse {
+  /// Token usage metadata for processing the generate content request.
```

Comment:
nit: Shouldn't internal line-based comments be two slashes?

Reply:
We don't seem to be very consistent about `//` vs. `///` for `internal` variables / constants (e.g., https://github.com/firebase/firebase-ios-sdk/blob/3783c8a1c3b5058cca47d81b20a903299b1f861f/FirebaseVertexAI/Sources/GenerativeModel.swift#L25-L47) but you pointed out a very important issue... these should be `public` (wasn't caught because the test uses `@testable import`) and should remain `///` for DocC. Adding `public` now.